sentry-core/src/api.rs

```diff
@@ -264,18 +264,21 @@ pub fn last_event_id() -> Option<Uuid> {
 
 /// Start a new session for Release Health.
 ///
-/// Returns a guard that ends the session on drop and thus determines the
-/// lifetime of the session.
-///
 /// # Examples
 ///
 /// ```
-/// let session = sentry::start_session();
+/// sentry::start_session();
 ///
-/// // capturing any event / error here will update the sessions `errors` count.
+/// // capturing any event / error here will update the sessions `errors` count,
+/// // up until we call `sentry::end_session`.
 ///
-/// drop(session);
+/// sentry::end_session();
 /// ```
-pub fn start_session() -> SessionGuard {
+pub fn start_session() {
```

Comment:
It would be great to denote that this is an experimental API and not ready for a high volume of sessions yet.